ntoskrnl/io/pnpmgr/pnpreport.c

```diff
@@ -223,6 +223,9 @@ IoReportDetectedDevice(
     if (DeviceObject && *DeviceObject)
     {
         Pdo = *DeviceObject;
+        // FIXME! What to do for initializing instancePath??
+        // Pdo->DeviceObjectExtension->DeviceNode->InstancePath;
+        RtlInitEmptyUnicodeString(&instancePath, NULL, 0);
```

Comment:
As far as I can see, Windows just returns `STATUS_NO_SUCH_DEVICE` if you pass a PDO without DeviceNode

Reply:
Hehe ok. Also this diff should not be part of this PR 😆